scripts/setup.sh

```diff
@@ -63,6 +63,9 @@ DC_DESTINATION=/usr/local/bin/docker-compose
 sudo curl -L https://github.com/docker/compose/releases/download/${DC_VERSION}/docker-compose-$(uname -s)-$(uname -m) -o $DC_DESTINATION
 sudo chmod 755 $DC_DESTINATION
 
+# Install Apache benchmark testing tool
```

Comment:
why are we installing tools for all the containers? This should be part of coverage script where we plan to use these tools.

Reply:
Adding it to the docker_test() of the functional_tests file instead. Works?